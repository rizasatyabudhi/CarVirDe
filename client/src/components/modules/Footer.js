import React, { Component } from "react";

export default class Footer extends Component {
  render() {
    return (
      <div>
        <footer class="page-footer deep-purple darken-3 lighten-1">
          <div class="container">
            <div class="row">
              <div class="col l6 s12">
                <h5 class="white-text">CarVirde</h5>
                <p class="grey-text text-lighten-4">Your Traveling Solution</p>
              </div>
              <div class="col l4 offset-l2 s12">
                <i className="material-icons">contact_phone</i>
                <h5
                  style={{
                    display: "inline-block",
                    marginLeft: "20px",
                    fontSize: "15px"
                  }}
                  class="white-text"
                >
                  021-7574829
                </h5>
                <ul>
                  {/*  <li>
                    <a class="grey-text text-lighten-3" href="#!">
                      Link 1
                    </a>
                  </li>
                  <li>
                    <a class="grey-text text-lighten-3" href="#!">
                      Link 2
                    </a>
                  </li>
                  <li>
                    <a class="grey-text text-lighten-3" href="#!">
                      Link 3
                    </a>
                  </li>
                  <li>
                    <a class="grey-text text-lighten-3" href="#!">
                      Link 4
                    </a>
                  </li>
                  */}
                </ul>
              </div>
              <div class="col l4 offset-l2 s12">
                <i className="material-icons">email</i>
                <h5
                  style={{
                    display: "inline-block",
                    marginLeft: "20px",
                    fontSize: "15px"
                  }}
                  class="white-text"
                >
                  contact-support@carvirde.com
                </h5>
                <ul />
              </div>
            </div>
          </div>
          <div class="footer-copyright">
            <div class="container">
              Â© 2017 CarVirde
              <a
                class="grey-text text-lighten-4 right"
                href="https://github.com/rizasatyabudhi/CarVirDe"
              >
                https://github.com/rizasatyabudhi/CarVirDe
              </a>
            </div>
          </div>
        </footer>
      </div>
    );
  }
}
