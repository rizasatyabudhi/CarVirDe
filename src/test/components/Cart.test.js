import React from "react";
import { shallow } from "enzyme";
import Footer from "../../components/modules/Footer";
import {
  CarCart,
  AttractionCart,
  AttractionPackageCart
} from "../../components/modules/CartCard";

test("should render CarCart", () => {
  const wrapper = shallow(<CarCart />);
  expect(wrapper).toMatchSnapshot();
});

test("should render AttractionCart", () => {
  const wrapper = shallow(<AttractionCart />);
  expect(wrapper).toMatchSnapshot();
});
test("should render AttractionPackageCart", () => {
  const wrapper = shallow(<AttractionPackageCart />);
  expect(wrapper).toMatchSnapshot();
});
