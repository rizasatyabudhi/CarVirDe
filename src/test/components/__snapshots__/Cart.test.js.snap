// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AttractionCart 1`] = `
<div
  class="col s12"
>
  <div
    class="card-panel hoverable"
  >
    <h5>
      Orderan Tempat Wisata
    </h5>
    <div
      className="row"
    >
      <div
        className="col s10"
      >
        <div
          className="card-content  "
        >
          <div
            className="row"
          >
            <div
              className="col s6"
            >
              <ul>
                <li>
                  <div
                    className="row"
                  >
                    <div
                      className="col s12"
                    >
                      <span
                        className="category"
                      >
                        Attraction Name :
                      </span>
                    </div>
                  </div>
                </li>
                <li>
                  <div
                    className="row"
                  >
                    <div
                      className="col s12"
                    >
                      <span
                        className="category"
                      >
                        Location :
                      </span>
                    </div>
                  </div>
                </li>
              </ul>
            </div>
            <div
              className="col s6"
              style={
                Object {
                  "marginTop": "20px",
                }
              }
            >
              <span
                className="category"
              >
                Tanggal Kedatangan: 
              </span>
              <input
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s2"
      >
        <h5>
          Rp 
        </h5>
      </div>
    </div>
  </div>
</div>
`;

exports[`should render AttractionPackageCart 1`] = `
<div
  class="col s12"
>
  <div
    class="card-panel hoverable"
    style={
      Object {
        "height": "470px",
      }
    }
  >
    <h5>
      Orderan Paket Wisata
    </h5>
    <div
      class="col s4"
    >
      <div
        class="card horizontal z-depth-3"
      >
        <div
          class="card-stacked"
        >
          <div
            class="card-content"
          >
            <div
              className="col s12"
            >
              <p
                style={
                  Object {
                    "fontWeight": "800",
                  }
                }
              />
            </div>
            <div
              className="row"
            >
              <div
                className="col s12"
              >
                <ul>
                  <li>
                    <i
                      className="material-icons"
                    >
                      location_on
                    </i>
                  </li>
                  <li>
                    <i
                      className="material-icons"
                    >
                      attach_money
                    </i>
                    <span>
                      Rp  
                      /Pax
                    </span>
                  </li>
                </ul>
              </div>
              <div
                className="col s12 paragraph"
                style={
                  Object {
                    "marginTop": "23px",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="col s4"
    >
      <div
        class="card horizontal z-depth-3"
      >
        <div
          class="card-stacked"
        >
          <div
            class="card-content"
          >
            <div
              className="col s12"
            >
              <p
                style={
                  Object {
                    "fontWeight": "800",
                  }
                }
              />
            </div>
            <div
              className="row"
            >
              <div
                className="col s12"
              >
                <ul>
                  <li>
                    <i
                      className="material-icons"
                    >
                      location_on
                    </i>
                  </li>
                  <li>
                    <i
                      className="material-icons"
                    >
                      attach_money
                    </i>
                    <span>
                      Rp  
                      /Pax
                    </span>
                  </li>
                </ul>
              </div>
              <div
                className="col s12 paragraph"
                style={
                  Object {
                    "marginTop": "23px",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="col s4"
    >
      <div
        class="card horizontal z-depth-3"
      >
        <div
          class="card-stacked"
        >
          <div
            class="card-content"
          >
            <div
              className="col s12"
            >
              <p
                style={
                  Object {
                    "fontWeight": "800",
                  }
                }
              />
            </div>
            <div
              className="row"
            >
              <div
                className="col s12"
              >
                <ul>
                  <li>
                    <i
                      className="material-icons"
                    >
                      location_on
                    </i>
                  </li>
                  <li>
                    <i
                      className="material-icons"
                    >
                      attach_money
                    </i>
                    <span>
                      Rp  
                      /Pax
                    </span>
                  </li>
                </ul>
              </div>
              <div
                className="col s12 paragraph"
                style={
                  Object {
                    "marginTop": "23px",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col s4"
      style={
        Object {
          "marginTop": "20px",
        }
      }
    >
      <span
        className="category"
      >
        Tanggal Kedatangan: 
      </span>
      <input
        type="text"
      />
    </div>
    <div
      className="col s12"
    >
      <h5
        className="right"
      >
        Rp 
      </h5>
    </div>
  </div>
</div>
`;

exports[`should render CarCart 1`] = `
<div
  class="col s12"
>
  <div
    class="card-panel hoverable hover"
  >
    <h5>
      Orderan Rental Mobil
    </h5>
    <div
      className="row"
    >
      <div
        className="col s10"
      >
        <div
          className="card-content  "
        >
          <div
            className="row"
          >
            <div
              className="col s6"
            >
              <ul>
                <li>
                  <div
                    className="row"
                  >
                    <div
                      className="col s12"
                    >
                      <span
                        className="category"
                      >
                        Tipe Mobil  :
                      </span>
                    </div>
                  </div>
                </li>
                <li>
                  <div
                    className="row"
                  >
                    <div
                      className="col s12"
                    >
                      <span
                        className="category"
                      >
                        Merk Mobil  :
                      </span>
                    </div>
                  </div>
                </li>
              </ul>
            </div>
            <div
              className="col s6"
            >
              <ul>
                <li>
                  <div
                    className="row"
                  >
                    <div
                      className="col s12"
                    >
                      <span
                        className="category"
                      >
                        Tahun     :
                      </span>
                    </div>
                  </div>
                </li>
                <li>
                  <div
                    className="row"
                  >
                    <div
                      className="col s12"
                    >
                      <span
                        className="category"
                      >
                        Transmisi  : 
                      </span>
                    </div>
                  </div>
                </li>
              </ul>
            </div>
            <div
              className="col s12"
            >
              <span
                className="category"
              >
                Pakai Supir: 
              </span>
              <input
                class="with-gap"
                id="yes"
                name="group"
                type="radio"
              />
              <label
                className="label-style"
                for="yes"
              >
                Ya
              </label>
              <input
                class="with-gap"
                id="no"
                name="group"
                type="radio"
              />
              <label
                className="label-style"
                for="no"
              >
                Tidak
              </label>
            </div>
            <div
              className="col s4"
              style={
                Object {
                  "marginTop": "20px",
                }
              }
            >
              <span
                className="category"
              >
                Tanggal Rental: 
              </span>
              <input
                type="text"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s2"
      >
        <h5>
          Rp 
        </h5>
      </div>
    </div>
  </div>
</div>
`;
